test/library/packages/Sort/RadixSort/radixMSB.good

```diff
@@ -0,0 +1,3 @@
+radixMSB.chpl:14: In function 'radix_sort_msd':
```

Comment:
Can we catch this with `--no-warnings` in a `radixMSB.compopts` file?

Reply:
thanks, I made a  radixMSB.compopts for the same and updated the PR.